cms/templates/studio_xblock_wrapper.html

```diff
@@ -128,14 +128,16 @@
     <div class="wrapper-xblock-message xblock-validation-messages" data-locator="${xblock.location | h}"/>
 % endif
 
-% if is_root or not xblock_url:
-    <article class="xblock-render">
-    ${content}
-    </article>
-% else:
-    <div class="xblock-message-area">
-    ${content}
-% endif
+% if xblock_context.get('show_preview', True):
+    % if is_root or not xblock_url:
+        <article class="xblock-render">
+        ${content}
+        </article>
+    % else:
+        <div class="xblock-message-area">
```

Comment:
Unrelated: I know we didn't change this just now, but this lone `<div>` really looks wrong to me... I think I've encountered problems caused by this before. But it may be matched up with another closing tag somewhere else? Still ugly.

Also above is a `<div />` which is confusing and only valid in XHTML but not HTML4 or 5 :-/